apps/rabbitmq_prelaunch/src/rabbit_boot_state.erl

```diff
@@ -0,0 +1,97 @@
+%%%-------------------------------------------------------------------
+%% The contents of this file are subject to the Mozilla Public License
+%% Version 1.1 (the "License"); you may not use this file except in
+%% compliance with the License. You may obtain a copy of the License
+%% at https://www.mozilla.org/MPL/
+%%
+%% Software distributed under the License is distributed on an "AS IS"
+%% basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See
+%% the License for the specific language governing rights and
+%% limitations under the License.
+%%
+%% The Original Code is RabbitMQ.
+%%
+%% The Initial Developer of the Original Code is GoPivotal, Inc.
+%% Copyright (c) 2007-2020 Pivotal Software, Inc.  All rights reserved.
+%%
+
+-module(rabbit_boot_state).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-export([get_boot_state/0,
+         set_boot_state/1,
+         wait_for_boot_state/1]).
```

Comment:
What do you think of renaming those functions to `get/0`, `set/1` and `wait_for/1`? The module name indicates the context of what is to be set/get/waited.

Reply:
seems reasonable to me since the exports of that module are small.

Reply:
It does mean that there is one place in this module where we call the function where it will become `?MODULE:get()` to avoid ambiguity with the bif get, but I think that is acceptable.

Reply:
I agree.